src/App.css

```diff
@@ -28,17 +28,24 @@ body {
   padding: 3% 4%;
 }
 
+.sticky {
+  position: sticky;
+  top: 0;
+  background: #fff;
+  z-index: 1;
+}
+
```

Comment:
- [ ] [OPTIONAL] Great job on your book store it looks almost identical to the Zeplin design however there are are a few things I spotted that are different from the design if you fix them your design fill be pixel perfect 👍 . Please consider making the following changes:

   - [ ] Please align the navbar links to the left instead of center just like in Zeplin design. 
   - [ ] Please use `#0290ff` for the profile icon. For that I would highly recommend using` react-icons`. React-icons is a small library that helps you add icons (from all different icon libraries) to your React apps. It delivers the icons to your app as components so they're easier to work with, and it lets you customise the style so they're consistent with the overall style of your app.